piet-direct2d/src/lib.rs

```diff
@@ -418,6 +388,55 @@ impl<'a> RenderContext for D2DRenderContext<'a> {
 }
 
 impl<'a> D2DRenderContext<'a> {
+    fn fill_impl(&mut self, shape: impl Shape, brush: &impl IntoBrush<Self>, fill_rule: FillRule) {
+        let brush = brush.make_brush(self, || shape.bounding_box());
+
+        // TODO: do something special (or nothing at all) for line?
```

Comment:
As mentioned, this comment could be changed from a TODO to an explanation why we're not handling lines.

Reply:
I figured theres a chance that it could be possible to turn lines into a noop, or something more efficient than the fallback, and if someone does something like me and draws 1000 lines that may end up being worthwhile.